Sources/Plasma/PubUtilLib/plNetTransport/plNetTransport.cpp

```diff
@@ -241,14 +241,14 @@ int plNetTransport::SendMsg(int chan, plNetMessage* netMsg) const
         {           
 //          if ((ncRet=nc->GetNetClientComm().SendMsg(netMsg, nc->GetServerPeerID(), sendFlags, msgSize)) != plNetCore::kNetOK)
             NetCommSendMsg(netMsg);
-            ret=0;  // sent
+            ret = 0; // sent
         }
     }
     else
     {
         hsStatusMessage("EMPTY TRANSPORT GROUP\n");
     }
-    return ret;
+    //return ret;
```

Comment:
It looks like there's still some unused return value stuff in this function.

Reply:
This entire code is dead actually, because the second line of the method is an unconditional `return`. I wasn't sure if the code was left in for a reason, so I just commented out the return for now, and wanted to take a better look through the network message code later.

Reply:
I'm holding off on approval until we see where this goes.

Reply:
Ah, by "later" I meant "probably much later as its own PR". Didn't want to change the network code much, so that significant logic changes don't get overlooked in between all the simple type renamings. But I can take out this dead code if you prefer - that should be safe enough.

Reply:
Yes, I think we want to see dead code deleted instead of commented out.